database_design.txt

```diff
@@ -0,0 +1,87 @@
+E-Commerce Database
+---------------------------
+This database contains customer table, shipping address table, 
+invoice table, payment table, product table, and order item table
+
+Keyword:
+PK --> Primary Key
+FK --> Foreign Key
+
+Customer Table
+Purpose: This table stores information about a customer
+
+Fields:
+    - customer id (PK)
+    - first name
+    - last name
+    - email address
```

Comment:
if the database engine supports (looks like MS SQL does), should we mark this as unique? so it's enforced that there is no 2 customers with the same email address